lib/Ohmbrewer_Temperature_Sensor.cpp

```diff
@@ -72,6 +79,22 @@ Ohmbrewer::TemperatureSensor::~TemperatureSensor() {
 }
 
 /**
+ * Sets the probeID
+ * @param - id char[] of the ds18b20 probe
+ */
+void Ohmbrewer::TemperatureSensor::setProbeId(char id[8]){
+    _probeId = id;
+}
+/**
+ * reads current probes connected to busPin
+ * @param - sensors array to update with ID values of all(max10) the connected ds18b20 probes
+ * @returns - number of sensors discovered
+ */
+int Ohmbrewer::TemperatureSensor::findProbeIds(uint8_t sensors[80]){
+    return (int)ow_search_sensors(10, sensors);
```

Comment:
i was casting it to int so that the callee could deal with it as a normal number.  

and search sensors returns num sensors but updates sensors with all the IDs